knowledge_graph/wikibase.py

```diff
@@ -946,3 +948,96 @@ async def add_alternative_labels(
         return await self.add_alternative_labels_async(
             wikibase_id, alternative_labels, language
         )
+
+    async def search_help_pages_async(self, search_term: str) -> list[str]:
+        """Async method to search for help pages in Wikibase."""
+        client = await self._get_client()
+        response = await client.get(
+            url=self.api_url,
+            params={
+                "action": "query",
+                "list": "search",
+                "srsearch": search_term,
+                "srnamespace": self.HELP_NAMESPACE,
+                "format": "json",
+            },
+        )
+        response.raise_for_status()
+        data = response.json()
+        return [result["title"] for result in data.get("query", {}).get("search", [])]
+
+    @async_to_sync
+    async def search_help_pages(self, search_term: str) -> list[str]:
+        """Sync wrapper for search_help_pages_async"""
+        return await self.search_help_pages_async(search_term)
+
+    async def get_help_page_async(self, page_title: str) -> str:
+        """Async method to get help page content as markdown."""
+        client = await self._get_client()
+        response = await client.get(
+            url=self.api_url,
+            params={
+                "action": "parse",
+                "page": page_title,
+                "format": "json",
+            },
+        )
+        response.raise_for_status()
+        data = response.json()
+        html_content = data.get("parse", {}).get("text", {}).get("*", "")
+        if not html_content:
+            return ""
+        markdown_content = html2text.html2text(html_content)
+        return markdown_content
+
+    @async_to_sync
+    async def get_help_page(self, page_title: str) -> str:
+        """Sync wrapper for get_help_page_async"""
+        return await self.get_help_page_async(page_title)
+
+    async def search_concepts_async(
+        self,
+        search_term: str,
+        limit: Optional[int] = None,
+        timestamp: Optional[datetime] = None,
```

Comment:
could you add descriptions of what these parameters mean? `timestamp` is the one that's unclear to me - but think it's worth adding to all given this is in a shared/production-y bit of code!